# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../exim/glibc-2.28.patch
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- exim-4.92/OS/Makefile-Linux.vanilla	2019-06-04 13:56:21.547638051 +0000
+++ exim-4.92/OS/Makefile-Linux	2019-06-04 13:56:30.471638014 +0000
@@ -19,7 +19,7 @@
 DBMLIB = -ldb
 USE_DB = yes
 
-LIBS = -lnsl -lcrypt -lm
+LIBS = -lcrypt -lm
 LIBRESOLV = -lresolv
 
 X11=/usr/X11R6
